coalib/coala.py

```diff
@@ -15,15 +15,13 @@
 
 from pyprint.ConsolePrinter import ConsolePrinter
 
-from coalib.coala_main import run_coala
 from coalib.misc.Exceptions import get_exitcode
 from coalib.collecting.Collectors import filter_capabilities_by_languages
 from coalib.output.ConsoleInteraction import (
     acquire_settings, nothing_done, print_results, print_section_beginning,
     show_bears, show_language_bears_capabilities)
```

Comment:
can't this import be deferred? At least acquire_settings isn't used and reimported later, can we import only what we need here?